app.js

```diff
@@ -11,60 +11,78 @@ const catchphraseInput = document.getElementById('catchphrase-input');
 const catchphraseButton = document.getElementById('catchphrase-button');
 
 // set state for how many times the user changes the head, middle, and bottom
+let headCount = 0;
+let middleCount = 0;
+let bottomCount = 0;
+
 // set state for all of the character's catchphrases
+let phraseArray = ['Welcome to your super perfect and realistic character customizer'];
+
+displayCatchphrases();
+
+displayStats();
```

Comment:
Same thing with this function call as above.

Reply:
I put those there intensionally because I wanted the default phrases and stats to be there without having to click "add slogan' button